vars/folioParameters.groovy

```diff
@@ -50,6 +50,10 @@ def namespace() {
     return _paramExtendedSingleSelect('NAMESPACE', 'CLUSTER', folioStringScripts.getNamespaces(), '(Required) Select cluster namespace')
 }
 
+def repository() {
```

Comment:
For now we use only 'platform-complete' repository as platform-core is not maintained that is why we didn't migrate this parameter. Any case if you deside add new parameter use SCREAMING_CASE for jenkins job parameters